data/mods/ssb/abilities.js

```diff
@@ -223,6 +223,19 @@ let BattleAbilities = {
 			}
 		},
 	},
+	// Elgino
+	giblovepls: {
+		desc: "This Pokemon's Defense is raised 1 stage and heals 20% after it is damaged by a contact move.",
+		shortDesc: "Defense +1 and heal 20% after hit by contact move.",
+		onAfterDamage(damage, target, source, effect) {
+			if (effect && effect.flags['contact']) {
+				this.boost({def: 1});
+				this.heal(target.maxhp / 5);
```

Comment:
Specify the pokemon to heal/boost

Reply:
What he means basically is just to add `, target` inside both. Or something like that

Reply:
Although, kung fu panda uses similar code and doesnt specify so that irks me a bit :V

Reply:
Then that needs to be updated too.